ynput/OpenPype
# Max: Remove creating tyCache Objects as export option

## Changelog Description
When users publish tycache and "Create tyCache Object if not Created" being enabled, the created tycache Object would laoad the newly published file straightly after publishing.

## Additional info
n/a

## Testing notes:
1. Create TyCache Family
2. Make sure "Create tyCache Object if not Created"  selected in the published tycache attributes 
![image](https://github.com/ynput/OpenPype/assets/64118225/e97c8875-9b14-4009-873b-2509ad2f1430)

3. Publish
